wurst/systems/core/Experience.wurst

```diff
@@ -96,7 +96,7 @@ init
                 let pos = troll.getPos()
                 let killer = ENUM_GROUP
                     ..enumUnitsInRange(pos, expRange, Filter(() -> begin
-                        return GetFilterUnit().getOwner().getId() < bj_MAX_PLAYER_SLOTS
+                        return GetFilterUnit().getOwner() != players[PLAYER_NEUTRAL_AGGRESSIVE] and GetFilterUnit().getOwner() != players[PLAYER_NEUTRAL_PASSIVE]
```

Comment:
Let's just do `GetFilterUnit().isType(UNIT_TYPE_HERO)`

Reply:
New meta to feed yourself to the One so it can kill the enemy fighting it.

Reply:
`GetFilterUnit().getOwner().getTroll() != null` would work, right?

Reply:
Probably